memeBot/memeBot.py

```diff
@@ -337,9 +352,21 @@ def privmsg(self, user, channel, message):
                 self.murica(channel, host, temp_time)
 
             # match rip
-            elif re.search(r"(\brip\b)", message.lower()):
```

Comment:
y no allow big "RIP"?

Reply:
Big RIP makes MemeBot respond whenever Rob's username is mentioned